standalone/nginx/default.conf

```diff
@@ -7,6 +7,9 @@ server {
     access_log /var/log/nginx/access_joinmarket_webui.log;
     error_log /var/log/nginx/error_joinmarket_webui.log;
 
+    auth_basic "";
```

Comment:
Is this a name that's shown to the user in the basic auth dialog? If so let's put in something human readable here instead of the empty string.

Reply:
Good idea. Do you have a suggestion?

Reply:
Using "JoinMarket WebUI". :+1: